recipes/buttons/callbar_button_with_popover/callbar_button_with_popover.vue

```diff
@@ -32,7 +32,7 @@
       :show-close-button="showCloseButton"
       padding="none"
       class="dt-recipe--callbar-button-with-popover--popover-wrapper"
-      :dialog-class="['dt-recipe--callbar-button-with-popover--popover', contentClass]"
+      :dialog-class="['dt-recipe--callbar-button-with-popover--popover d-mb12', contentClass]"
```

Comment:
Like tooltip, you may also use offset on dt-popover, is that what you're looking to do?

Reply:
Yes! offset is better, updated.

<img width="545" alt="Screen Shot 2023-04-14 at 3 24 21 PM" src="https://user-images.githubusercontent.com/70488122/232164816-14f9c5cc-cc8a-405c-88c9-96d49db04b08.png">